src/tribler/core/components/tests/test_base_component.py

```diff
@@ -141,3 +122,36 @@ class ComponentB(Component):
         assert isinstance(component_b, NoneComponent)
         assert isinstance(component_b.any_attribute, NoneComponent)
         assert isinstance(component_b.any_attribute.any_nested_attribute, NoneComponent)
+
+
+def test_get_instance_direct_match(tribler_config: TriblerConfig):
+    class AncestorOfB(ComponentB):
```

Comment:
- Since this is a subclass, It would better to rename it to something like `DerivedB` or `ChildrenOfB` or `SubClassOfB`. I consider ancestor is not be a right word.
- Same applies for other examples in this test file.

Reply:
Thank you for the suggestion. Applied.